doc/book/src/user/release-support.md

```diff
@@ -42,8 +42,7 @@ time, and may shift due to changes in network solution power.
 <!-- RELEASE_SCRIPT_START_MARKER - If you make changes here, check make-release.py -->
 | `zcashd` version | Release date | Halt height | End of Support |
 | ---------------- | ------------ | ----------- | -------------- |
-| 5.9.0-rc1 | 2024-04-17 | 2603900 | 2024-08-07 |
-| 5.9.0 | 2024-04-19 | 2606000 | 2024-08-09 |
 | 5.9.1-rc1 | 2024-05-22 | 2643624 | 2024-09-11 |
 | 5.9.1 | 2024-05-23 | 2644900 | 2024-09-12 |
+| 5.10.0-rc1 | 2024-08-22 | 2700600 | 2024-10-31 |
```

Comment:
I checked that this halt height is before the halving height of 2726400. Specifically, it is ~22.4 days before the halving at the expected block rate. I also checked that it corresponds to the changes in `src/deprecation.h`.